setup.py

```diff
@@ -11,7 +11,7 @@
     "check_shapes>=1.0.0",
     "deprecated",
     "multipledispatch>=0.6",
-    "numpy",
+    "numpy<2",
```

Comment:
Out of curiosity, why is this necessary?

Reply:
This should have already been there. We don't (yet) support numpy 2 — which came out a few months ago — in any of our toolboxes.